client/src/views/admin/AdminList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <b-container>
+    <table class="table">
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Name</th>
+          <th scope="col">Role</th>
+          <th scope="col">Email</th>
+          <th scope="col">Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(adminDetails, index) in admins" :key="index">
+          <td>{{ index + 1 }}</td>
+          <td>{{ adminDetails.name }}</td>
+          <td>{{ adminDetails.role }}</td>
+          <td>{{ adminDetails.email }}</td>
+          <td>
+            <button @click="removeAdmin(adminDetails.id)">Remove</button>
+            <button @click="disableAdmin(adminDetails.id)">Disable</button>
+            <button @click="verifyAdmin(adminDetails.id)">Verify</button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </b-container>
+</template>
+
+<script>
+export default {
+  name: "AdminList",
+  data() {
+    return { admins: [], root: process.env.VUE_APP_ROOT_API };
+  },
+  mounted() {
+    this.fetchAdminsList();
+  },
+  methods: {
+    fetchAdminsList() {
+      this.$axios
+        .get(`${this.root}/admin`)
+        .then((response) => {
+          if (response.data.status) {
+            this.admins = response.data.data;
```

Comment:
pls check in UI, because of adding count , response.data.data might not contain admin list

Reply:
Updated